README.md

```diff
@@ -1,128 +1,201 @@
-# sneaky-package 🥸  
-A ready-to-deploy Python package designed to **stealthily integrate files** within a machine, ensuring discreet and seamless file operations without detection. This project is nothing more than a *lightweight & faster* Python package that aims to highlight and improve security by demonstrating potential vulnerabilities in the system when connected with **`pip`**.
+# SNEAKY PACKAGE 👾
+![GSSoC-ext](GSSoC-Ext.png)
+## Table of Contents
+1. [🚀Project Overview](#project-overview)
+2. [✨ Key Features ✨](#✨key-features✨)
+3. [⛓️ What is a **Pip Package**? ⛓️](#⛓️what-is-a-pip-package-⛓️)
+4. [🪪 Licence](#🪪licence)
+5. [🚀 How to Contribute to This Project](#🚀-how-to-contribute-to-this-project)
+   - [🍴 Fork the Repository](#🍴-fork-the-repository)
+   - [💻 Clone Your Fork](#💻-clone-your-fork)
+   - [🌿 Create a New Branch](#🌿-create-a-new-branch)
+   - [🛠️ Make Your Changes](#🛠️-make-your-changes)
+   - [✅ Finalize Your Changes](#✅-finalize-your-changes)
+   - [💬 Commit Your Changes](#💬-commit-your-changes)
+   - [📤 Push Your Changes](#📤-push-your-changes)
+   - [🔄 Create a Pull Request (PR)](#🔄-create-a-pull-request-pr)
+   - [🔎 Review Changes](#🔎-review-changes)
+6. [🏆 Contribution Points](#🏆-contribution-points)
+7. [📄 GSSoC Guidelines 📄](#📄-gssoc-guidelines-📄)
+8. [📑 Note for Contributors](#📑-note-for-contributors)
+9. [Technologies Used](#technologies-used)
+10. [Contributors ✨](#contributors-✨)
+11. [💌 Ending Note](#💌-ending-note)
+
+## 🚀Project Overview
+
+A ready-to-deploy Python package designed to stealthily integrate files within a machine, ensuring discreet and seamless file operations without detection. This project is nothing more than a lightweight & faster Python package that aims to highlight and improve security by demonstrating potential vulnerabilities in the system when connected with pip.
 
 ---
 
-## Getting Started! 🚀
+## ✨ Key Features ✨
+
+### 💾 Designed to Look Conventional
```

Comment:
I would suggest concise points and shorter description. Key features must be noticeable on first glance, one doesn't go reading them.